dexs/extended/index.ts

```diff
@@ -0,0 +1,41 @@
+import fetchURL from "../../utils/fetchURL"
+import { FetchResultVolume, SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import {getUniqStartOfTodayTimestamp} from "../../helpers/getUniSubgraphVolume";
+
+const historicalVolumeEndpoint = (dateTime: string) => `https://api.prod.x10.exchange/api/v1/exchange/stats/trading?fromDate=${dateTime}&toDate=${dateTime}`
```

Comment:
why fromDate eq toDate ?

Reply:
That's the way to request data for 1 day